src/main/scala/org/clulab/reach/context/ContextEngine.scala

```diff
@@ -66,30 +67,30 @@ object ContextEngine {
   def getDescription(mention:BioMention, voc:Map[(String, String), String]):String = {
     val key = getContextKey(mention)
     if(key._2.startsWith("uaz:")){
-      println(s"Warning: ${mention.text}")
+      logger.debug(s"Warning: ${mention.text}")
```

Comment:
I believe these should also be a different level.  Please coordinate with @hickst on these.